### ISSUE TITLE
CI: ping xip.io fails too often

### ISSUE THREAD

**Issue by alexkalmuk** [embox/embox#1865]
`ping 192.168.128.128.xip.io` fails too often for both expect tests and start_script:

https://github.com/embox/embox/runs/505471758#step:10:266 (start_script)
https://github.com/embox/embox/runs/505907968#step:10:208 (expect)

**Comment by AntonKozlov** [comment]
Have you looked at the source of the error? I had no chance to look at.

```
ping: no error 192.168.128.128.xip.io
ping: Command returned with code -22: Invalid argument
error: Unknown error code. PLEASE REPORT ISSUE 519!
```

It fails from time to time and the error message is really suspicious (`no error`). It really looks like a problem with DNS resolve, misreported. And I haven't seen the error on Travis, right? It may mean embox issue reproducing only in certain environments

**Comment by alexkalmuk** [comment]
No, I haven't look in more detail yet, but I have very similar observations.

**Comment by alexkalmuk** [comment]
@AntonKozlov 

> And I haven't seen the error on Travis, right?

https://travis-ci.org/github/embox/embox/jobs/662978056#L9029
